### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by Steffy-zxf**
add api reference

### CODE REVIEW

File: modules/text/sentiment_analysis/senta_bow.bak/test.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# coding:utf-8
```
Comment (nepeplwu) [inline modules/text/sentiment_analysis/senta_bow.bak/test.py]: senta_bow.bak?

File: modules/text/text_review/porn_detection_lstm/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# PornDetectionGRU API说明
+
+## detection(texts=[], data={}, use_gpu=False, batch_size=1)
+
+porn_detection_gru预测接口，鉴定输入句子是否为黄雯
```
[the remark this answers is not in the document]
Comment (Steffy-zxf) [reply modules/text/text_review/porn_detection_lstm/README.md]: done

File: modules/text/text_review/porn_detection_lstm/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+**参数**
+
+* texts(list): 待预测数据，如果使用texts参数，则不用传入data参数，二选一即可
+* data(dict): 预测数据，key必须为text，value是带预测数据。如果使用data参数，则不用传入texts参数，二选一即可。建议使用texts参数，data参数后续会废弃。
+* use_gpu(bool): 是否使用GPU预测
```
[the remark this answers is not in the document]
Comment (Steffy-zxf) [reply modules/text/text_review/porn_detection_lstm/README.md]: done

File: modules/text/text_review/porn_detection_lstm/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# PornDetectionGRU API说明
```
[the remark this answers is not in the document]
Comment (Steffy-zxf) [reply modules/text/text_review/porn_detection_lstm/README.md]: done

File: modules/text/text_review/porn_detection_lstm/test.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        if six.PY2:
+            print(json.dumps(
+                results[index], encoding="utf8", ensure_ascii=False))
+        else:
+            print(results[index])
+
+    results = porn_detection_lstm.detection(
+        texts=test_text, use_gpu=False, batch_size=2)
+    for index, result in enumerate(results):
+        if six.PY2:
```
[the remark this answers is not in the document]
Comment (Steffy-zxf) [reply modules/text/text_review/porn_detection_lstm/test.py]: 预测接口不会判断，命令行预测会判断

File: modules/text/sentiment_analysis/senta_lstm/test.py [hunk 6]
```diff
@@ -7,20 +7,29 @@
...
 
 if __name__ == "__main__":
-    # Load Senta_Bow Module
-    senta = hub.Module(name='senta_lstm')
-    test_text = ["这家餐厅很好吃", "这部电影真的很差劲"]
-    senta.context()
-    results = senta.sentiment_classify(data={'text': test_text}, use_gpu=True)
-    # execute predict and print the result
+    # Load porn_detection module
+    porn_detection_lstm = hub.Module(name="porn_detection_lstm")
```
[the remark this answers is not in the document]
Comment (Steffy-zxf) [reply modules/text/sentiment_analysis/senta_lstm/test.py]: done

File: modules/text/sentiment_analysis/senta_bow.bak/test.py (hunk 1 above)
Comment (Steffy-zxf) [reply modules/text/sentiment_analysis/senta_bow.bak/test.py]: done

**Pull request merged**